file-issue-nine.js

```diff
@@ -0,0 +1 @@
+console.log('issue nine');
```

Comment:
review comment 1

Reply:
review_comment2_by_developer

Reply:
hello I am developer

Reply:
think_this_is_a pull_request_review_comment.created

Reply:
comment for event pull_request_review_comment.created

Reply:
New PULL_REVIEW comment